ssl/t1_lib.c

```diff
@@ -1371,6 +1371,38 @@ int ssl_add_clienthello_tlsext(SSL *s, WPACKET *pkt, int *al)
         return 0;
     }
 
+    if (!SSL_IS_DTLS(s) && s->version >= TLS1_3_VERSION) {
```

Comment:
hm.  I think this is the third time I've seen that kind of if expression ...  macro?